### ISSUE TITLE
Get rid of Maria DB since its no longer supported in vitess

### ISSUE THREAD

**Issue by rsajwani** [vitessio/vitess#12166]

**Comment by deepthi** [comment]
Is this a placeholder issue? The title is factually incorrect because Data imports using MoveTables etc. from MariaDB source databases are still supported though actual support for running Vitess on MariaDB has indeed been deprecated
Ideally the issue would list the tasks remaining in terms of deleting unused MariaDB related code.

**Comment by rsajwani** [comment]
Yeah my Idea was to fill it with proper wordings later on .. I just wanted to put something so it doesn't fall through the cracks.

**Comment by rsajwani** [comment]
I will add the code location shortly.
